examples/src/main/python/gpu/nubma_example.py

```diff
@@ -0,0 +1,32 @@
+# adapted from https://www.slideshare.net/continuumio/gpu-computing-with-apache-spark-and-python
+
+import numpy as np 
+from numba import cuda, vectorize
+from pyspark.sql import SparkSession
+
+@cuda.jit('(float32[:], float32[:])')
+def func(inp, out):
+    i = cuda.grid(1)
+    if i < out.size:
+        out[i] = inp[i] ** 2
+
+
+def gpu_work(xs):
```

Comment:
would be cool if there was a CPU equivalent to compare with.

Reply:
updated to add a comparison